service/cloudfront/integ_test.go

```diff
@@ -1,5 +1,6 @@
 // Code generated by private/model/cli/gen-api/main.go. DO NOT EDIT.
 
+//go:build go1.15 && integration
```

Comment:
Should this tag addition for generated files be handled explicitly in `private/model/cli/gen-api/main.go`

Reply:
The generator running with the latest version of Go (1.17) will auto back fill in the missing `go:build` flags. We can either extend this PR, or do as a separate change since gofmt will provide backfilling.

Reply:
Actually given the test failure cases looks like this should be a part of the generator in this PR